Configurations/10-main.conf

```diff
@@ -6,16 +6,19 @@ my $vc_win64a_info = {};
 sub vc_win64a_info {
     unless (%$vc_win64a_info) {
         if (`nasm -v 2>NUL` =~ /NASM version ([0-9]+\.[0-9]+)/ && $1 >= 2.0) {
-            $vc_win64a_info = { as        => "nasm",
-                                asflags   => "-f win64 -DNEAR -Ox -g",
-                                asoutflag => "-o" };
+            $vc_win64a_info = { AS        => "nasm",
+                                ASFLAGS   => "-g",
+                                asflags   => "-Ox -f win64 -DNEAR",
+                                asoutflag => "-o " };
         } elsif ($disabled{asm}) {
-            $vc_win64a_info = { as        => "ml64",
+            $vc_win64a_info = { AS        => "ml64",
+                                ASFLAGS   => "",
                                 asflags   => "/c /Cp /Cx /Zi",
```

Comment:
BTW, /Zi is "Add symbolic debug info". You kind of suggest to keep it overridable...

Reply:
> You kind of suggest to keep it overridable...

Are you saying it should be overridable?

Reply:
Done